lib/ansible/plugins/callback/actionable.py

```diff
@@ -32,6 +32,7 @@ def __init__(self):
         self.super_ref.__init__()
         self.last_task = None
         self.shown_title = False
+        self.diff = False
```

Comment:
the correct fix here is changing super to `CallbackModule_default` instead of `CallbackModule`, then the `self.diff` and any future changes in the init will be inherited